src/Magnum/Trade/PhongMaterialData.h

```diff
@@ -124,6 +124,49 @@ class MAGNUM_TRADE_EXPORT PhongMaterialData: public AbstractMaterialData {
          */
```

Comment:
At the top of the class definition, there's an enum marked with `/** @todo what is this for?! */`. It was probably an initial attempt at this. Can you delete that thing?